src/lib.rs

```diff
@@ -14,14 +14,20 @@ use zkevm_opcode_defs::ISAVersion;
 use zkevm_opcode_defs::LogOpcode;
 use zkevm_opcode_defs::Opcode as Variant;
 
-pub fn run_program(bin_path: &str) -> U256 {
+pub fn run_program(bin_path: &str) -> (U256, VMState) {
```

Comment:
Why did you add VMState as a return value? Its never used anywhere

Reply:
I though it would be useful to inspect the VM's final state when testing.
Let's say we want to test if an instruction modifies the "eq" flag, I can't come up with another way to test it.

Reply:
Agree that being able to inspect the `VMState` is useful for testing, it comes in handy for the (yet to be pushed) PR for [arithmetic instructions](https://github.com/matter-labs/zksync-era/blob/main/docs/specs/zk_evm/vm_specification/zkSync_era_virtual_machine_primer.md#arithmetic-instructions).